projects/packages/my-jetpack/_inc/data/types.ts

```diff
@@ -36,3 +36,74 @@ export type WP_Error = {
 		status: number;
 	};
 };
+
+export type Purchase = {
```

Comment:
@robertsreberski looks like you were right that we should've kept this 😅

Reply:
😁